server/resource_server/emails/sender/mailer_send_service.go

```diff
@@ -0,0 +1,71 @@
+package sender
+
+import (
+	"context"
+	"fmt"
+	"github.com/mailersend/mailersend-go"
+	"globe-and-citizen/layer8/server/resource_server/models"
+	"net/http"
+	"time"
+)
+
+const emailSendTimeout = time.Second * 10
+
+type MailerSendService struct {
```

Comment:
create an interface here, sometimes in future we might need to mock the mailersendservice